src/test/java/test/listeners/issue1777/IssueTest.java

```diff
@@ -0,0 +1,61 @@
+package test.listeners.issue1777;
+
+import org.testng.ITestResult;
+import org.testng.TestNG;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.internal.RuntimeBehavior;
+import org.testng.xml.XmlSuite;
+import test.SimpleBaseTest;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IssueTest extends SimpleBaseTest {
+
+  public static final String GITHUB_1777 = "GITHUB-1777";
+
+  @BeforeMethod
+  public void setup(ITestResult result) {
+    if (GITHUB_1777.equalsIgnoreCase(result.getMethod().getDescription())) {
+      System.setProperty(RuntimeBehavior.TESTNG_LISTENERS_ALWAYSRUN, Boolean.TRUE.toString());
+    }
```

Comment:
In case something was wrong before:
```java
} else {
System.setProperty(RuntimeBehavior.TESTNG_LISTENERS_ALWAYSRUN, Boolean.FALSE.toString());
} 
```